app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  def edit
+    @user = current_user
+    @data = params[:user]
+    if(@data != nil)
+      @new_email = @data["new_email"]
+      if(!@new_email.to_s.empty?)
+        if(@new_email.match(/[a-zA-Z0-9._%]@(?:[a-zA-Z0-9]+\.)[a-zA-Z]{2,4}/))
```

Comment:
Tev nevajag new_email. Lai lietotajs maina savu esošo.

Tev vnk vajag šo validaciju ielīkt modelī lai saglabašanas laikā ta izšautu. Paskaties googlā kā parasti to darā iekš Rails un ja gribi, vari pat pamklet gatavo gemu.